launch/tier4_perception_launch/launch/perception.launch.xml

```diff
@@ -31,6 +31,10 @@
   <arg name="centerpoint_model_name" default="centerpoint_tiny" description="options: `centerpoint` or `centerpoint_tiny`"/>
   <arg name="centerpoint_model_path" default="$(var data_path)/lidar_centerpoint"/>
 
+  <!-- PointPainting model parameters -->
+  <arg name="pointpainting_model_name" default="pointpainting" description="options: `pointpainting`"/>
```

Comment:
Can we use `lidar_detection_model` instead of adding new `pointpainting_model_name` arg?

Reply:
Sure. Does this meet your request? https://github.com/autowarefoundation/autoware.universe/pull/6106/commits/345ed3f09f8f3ec34ee08ab763e798fd390d76f1